### PR TITLE
Add money columns detection

### PR DESCRIPTION
**Pull request by Rob192**
This features enables to detect the columns that contain money based on the column label. In order to keep things separated, the output returned is separated into the findings based on the fields of the csv on one side, and the finding based on the labels on the other side. The merging between the two output is based on a AND gate, but if the test was done only based on the label or only based on the field, then the value is kept. This will allow reinforcing validation of the column content based on testing the label and on the fields.

### CODE REVIEW

File: analyse_csv.py
Comment (geoffreyaldebert) [inline analyse_csv.py:135]: I think this file is not needed in csv-detective python package. I'm not sure we have to keep it here. WDYT @psorianom ?

File: csv_detective/detect_fields/other/float_field/__init__.py
Comment (geoffreyaldebert) [inline csv_detective/detect_fields/other/float_field/__init__.py:19]: I would prefer for folder name : "float" like "booleen" example. We are on "Detect fields" folder so there is no ambiguity for me if we rename it only "float"

File: csv_detective/detect_labels/other/money/check_col_name.py
Comment (geoffreyaldebert) [inline csv_detective/detect_labels/other/money/check_col_name.py:6]: No problem for me but as we discussed, you will have some errors with this check. For instance, header "pays d'europe" would be associated with money as "euro" is contained in it. But could be improved later.

File: csv_detective/explore_csv.py
Comment (geoffreyaldebert) [inline csv_detective/explore_csv.py:142]: Why don't retrieve a file furnished in argument with sys.argv ?

File: tests/test_csv.py
Comment (geoffreyaldebert) [inline tests/test_csv.py:14]: To change with a filepath on repo. Here : "./annuaire-de-leducation.csv'

**Review by geoffreyaldebert — COMMENTED** [review]
Well done @Rob192 It is ok for if you take care of the few remarks I have. We can discuss it tomorrow IRL.

File: csv_detective/detect_fields/other/float_field/__init__.py
Comment (Rob192) [reply csv_detective/detect_fields/other/float_field/__init__.py:19]: ok will do

File: analyse_csv.py
Comment (Rob192) [reply analyse_csv.py:135]: ok deleted !

File: csv_detective/detect_fields/other/float_field/__init__.py
Comment (Rob192) [reply csv_detective/detect_fields/other/float_field/__init__.py:19]: renamed to 'floats' to avoid confusion with python 'float' usages

File: csv_detective/explore_csv.py
Comment (Rob192) [reply csv_detective/explore_csv.py:142]: OK i just removed the main

File: tests/test_csv.py
Comment (Rob192) [reply tests/test_csv.py:14]: done

**Review by geoffreyaldebert — APPROVED** [review]

**Comment by geoffreyaldebert** [comment]
Approved. You just have to think about manage conflict on explore_csv before merge but for me it's ok. 
Waiting for @psorianom approval.

File: csv_detective/detect_fields/other/floats/__init__.py
Comment (psorianom) [inline csv_detective/detect_fields/other/floats/__init__.py:10]: pour être consistent avec le reste du code, j'ai enlèverai ce `main` d'ici

File: csv_detective/detect_labels/other/money/__init__.py
Comment (psorianom) [inline csv_detective/detect_labels/other/money/__init__.py:13]: pareil ici, je crois qu'on doit pas laisser des `mains` un peu partout.

File: csv_detective/detect_labels/other/money/check_col_name.py
Comment (psorianom) [inline csv_detective/detect_labels/other/money/check_col_name.py:9]: idem as before : block `main`

File: csv_detective/utils.py
Comment (psorianom) [inline csv_detective/utils.py:87]: celui est un commentaire ? on pourrait l'enlever ?

File: csv_detective/utils.py
Comment (psorianom) [inline csv_detective/utils.py:40]: These `test_fields` and `test_labels` functions have the same content. How are they different ?

File: csv_detective/explore_csv.py
Comment (Rob192) [reply csv_detective/explore_csv.py]: done

File: csv_detective/detect_labels/other/money/__init__.py
Comment (Rob192) [reply csv_detective/detect_labels/other/money/__init__.py]: done

File: csv_detective/detect_labels/other/money/check_col_name.py
Comment (Rob192) [reply csv_detective/detect_labels/other/money/check_col_name.py]: done

File: csv_detective/detect_fields/other/floats/__init__.py
Comment (Rob192) [reply csv_detective/detect_fields/other/floats/__init__.py]: done

File: csv_detective/utils.py
Comment (Rob192) [reply csv_detective/utils.py:87]: done

File: csv_detective/utils.py
Comment (Rob192) [reply csv_detective/utils.py]: you are right ! thanks. DONE

**Review by psorianom — APPROVED** [review]
seems legit! Merci @Rob192

**Pull request merged**
